www/front_src/src/Header/RessourceStatusCounter/index.tsx

```diff
@@ -51,6 +52,8 @@ const RessourceStatusCounter = <
       })
       .catch((error) => {
         if (error.response && error.response.status === 401) {
+          setIsAllowed(false);
```

Comment:
Same remark as above

Reply:
Thank's , changes made and i check if it's still ok to push again